src/command_modules/azure-cli-botservice/HISTORY.rst

```diff
@@ -3,6 +3,19 @@
 Release History
 ===============
 
+1.0.0
```

Comment:
1.0.0 is a taboo version number. This should be 0.2.0 or 2.0.0.  Given the sheer number of breaking changes, it is not appropriate to declare the module GA and I would recommend 0.2.0.

Reply:
Done, this PR is ready!